lib/services/amazonka-certificatemanager/gen/Amazonka/CertificateManager/AddTagsToCertificate.hs

```diff
@@ -128,9 +128,14 @@ instance Core.AWSRequest AddTagsToCertificate where
   response =
     Response.receiveNull AddTagsToCertificateResponse'
 
-instance Prelude.Hashable AddTagsToCertificate
+instance Prelude.Hashable AddTagsToCertificate where
+  hashWithSalt initialSalt AddTagsToCertificate' {..} =
+    initialSalt `Prelude.hashWithSalt` tags
```

Comment:
Looks like we're not hashing `certificateArn`? Is that right?

Reply:
It's a deliberate compilation performance optimisation! (Will fix, thanks for catching.)

Reply:
`const 0` is a very fast valid hash function.